build.sbt

```diff
@@ -5,6 +5,7 @@ pgpSecretRing := file(s"$gpgFolder/secring.gpg")
 lazy val commonDependencies: Seq[ModuleID] = Seq(
   %%("cats-core"),
   %%("freestyle-async"),
+  %%("shapeless"),
```

Comment:
Shapeless should already be transitively available through the dependency on `freestyle` since the core depends on it.